config/initializers/doorkeeper.rb

```diff
@@ -18,6 +18,8 @@ def self.optional
 
   realm "Panoptes"
 
+  grant_flows ["authorization_code", "client_credentials", "implicit", "password"]
```

Comment:
Do we have to explicitly declare the flows we are using now? Also are we using all these?